File: app/src/main/java/com/example/acplite/utilidades/UtilidadesEvento.java
package com.example.acplite.utilidades;

public class UtilidadesEvento {
    public static final String DB_NAME = "ACPLiteDB";
    public static final String EVENTS_TABLE_NAME = "Eventos";
    public static final String CAMPO_NOMBRE = "nombre";
    public static final String CAMPO_FECHA = "fecha";

    public static final String CREATE_TABLE_EVENT = "CREATE TABLE " + EVENTS_TABLE_NAME + " ( " +
            CAMPO_NOMBRE + " TEXT, " +
            CAMPO_FECHA + " TEXT)";

    public static final String UPDATE_TABLE_EVENT = "DROP TABLE IF EXISTS " + EVENTS_TABLE_NAME;
}

File: app/src/main/java/com/example/acplite/utilidades/UtilidadesEvento.kt
package com.example.acplite.utilidades

object UtilidadesEvento {
    const val DB_NAME = "ACPLiteDB"
    const val EVENTS_TABLE_NAME = "Eventos"
    const val CAMPO_NOMBRE = "nombre"
    const val CAMPO_FECHA = "fecha"
    const val CREATE_TABLE_EVENT = "CREATE TABLE " + EVENTS_TABLE_NAME + " ( " +
            CAMPO_NOMBRE + " TEXT, " +
            CAMPO_FECHA + " TEXT)"
    const val UPDATE_TABLE_EVENT = "DROP TABLE IF EXISTS " + EVENTS_TABLE_NAME
}